package exceptionHandling;

public class TryCatchException1 {
public static void main(String []args)


{
int i=90;
int j=0;
try {
	int x=i/j;
	
}
catch (Exception e)
{
	System.out.println((i+j)*2);
}
}
}
